docs/supported-tools.rst

```diff
@@ -0,0 +1,246 @@
+   But **when support for a tool or dependency ends, new builds may fail**.
+
+.. contents:: Contents
+   :local:
+   :depth: 3
+
+End of support policy
+---------------------
+
+Our policy defines how long a given tool or dependency is considered supported.
+After it reaches its end of support date,
+we don't offer support for builds/docs using these tools or dependencies,
+you'll need to upgrade in order to receive support.
+For :doc:`/commercial/index` we provide an extended support of six months after the official end of support date.
+
+For tools that define an EOL date, we try to follow that date for our policy.
+For tools that release their versions incrementally without an EOL date,
+we choose a date based on: the release of mayor versions (versions with breaking changes),
+the date since it was last updated, and its usage on our platform.
+
+.. note::
+
+   Some recent versions may be supported, but aren't listed on these tables
+   (specially minor updates from supported versions),
+   contact us if you have doubts.
+
+Default dependencies
+--------------------
+
+We install some dependencies in order to build your project without specifying their dependencies.
+In the past we used to set these dependencies to a specific version and update them after some time,
+but by doing so would break some builds and new projects wouldn't be able to use a new version by default.
+For this reason we are now installing their latest version (or latest supported version) by default.
+
+.. note::
+
+   In order to keep your builds reproducible,
+   it's highly recommended declaring its dependencies and versions explicitly.
+
+   .. TODO: link to this guide once it's written https://github.com/readthedocs/readthedocs.org/issues/7852.
+
+External dependencies (Python)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Python environment:
+
+Sphinx:
+  Projects created before Oct 20, 2020 use ``1.8.x``.
+  New projects use the latest version.
+
+Mkdocs:
+  Projects created before April 3, 2019 (April 23, 2019 for :doc:`/commercial/index`) use ``0.17.3``.
+  New projects use the latest version.
+
+sphinx-rtd-theme:
+  Projects created before October 20, 2020 (January 21, 2021 for :doc:`/commercial/index`) use ``0.4.3``.
+  New projects use the latest version.
+
+pip:
+  Latest version by default.
+
+setuptools:
+  Latest version by default.
+
+mock:
+  ``1.0.1`` (could be removed in the future).
+
+pillow:
+  ``5.4.1`` (could be removed in the future).
+
+alabaster:
+  ``0.7.x`` (could be removed in the future).
+
+commonmark:
+  ``0.8.1`` (could be removed in the future).
+
+recommonmark:
+  ``0.5.0`` (could be removed in the future).
+
+External dependencies (Conda)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Conda environment:
+
+Conda:
+   Miniconda2 ``4.6.14``
+   (could be updated to use the latest version by default).
```

Comment:
I think we can start sampling more projects to use `update_conda_at_startup`, we have noticed that this solves some OOM errors, and it's faster :)

Reply:
Yeah. However, we have found also that sometimes a conda upgrade broke the build. So, there is some tradeoff here.

I think the best solution here is to continue the path towards `mamba`.

Reply:
I don't think we should push to mamba as a replacement for conda, it should be an option. We already update pip on each build, if conda breaks is probably temporary and not so usual.

Reply:
It's true we update `pip`. However, `pip` is way more stable than `conda` in our experience :smile:

Reply:
Another difference between upgrading pip and conda is that conda takes _a lot_ of time to update.